style.css

```diff
@@ -0,0 +1,32 @@
+* {
+ margin: 0;
+ padding: 0;
+ width: 100%;
+ box-sizing: border-box;
+ border-style: 1px solid black;
+}
+
+html, body {
+ box-sizing: border-box;
+ border-style: 1px solid black;
+ height: 100%;
+ width: 100%;
+}
+
+.fa {
+ float: right;
+ font-size: 3em;
+}
+
+.main-nav {
+  padding: 0.1em;
+  font-size: 1.5em;
+  float:left;
+  width: 960px;
+}
+
+@media screen and (min-width:640px) {
+  .fa {
```

Comment:
this is only good if you're only using the one icon, otherwise all of your font awesome icons will disappear when the screen becomes wider. You can target this more specifically with just ".fa-bars"